File: android-app/src/main/java/fr/cph/chicago/entity/dto/FavoritesDTO.java
package fr.cph.chicago.entity.dto;

import java.util.List;

import fr.cph.chicago.entity.BikeStation;
import lombok.Builder;
import lombok.Data;

@Builder
@Data
public class FavoritesDTO {
    private TrainArrivalDTO trainArrivalDTO;
    private BusArrivalDTO busArrivalDTO;

    private boolean bikeError;
    private List<BikeStation> bikeStations;
}

File: android-app/src/main/java/fr/cph/chicago/entity/dto/FavoritesDTO.kt
package fr.cph.chicago.entity.dto

import fr.cph.chicago.entity.BikeStation

class FavoritesDTO(
    val trainArrivalDTO: TrainArrivalDTO? = null,
    val busArrivalDTO: BusArrivalDTO? = null,
    val bikeError: Boolean = false,
    val bikeStations: List<BikeStation>? = null)

